src/theme/mod.rs

```diff
@@ -20,6 +20,8 @@ pub struct Options {
 
     pub colour_scale: ColorScaleOptions,
 
+    pub embed_hyperlinks: EmbedHyperlinks,
```

Comment:
why do you need this, and please adapt the powertests .yaml to make the change, and add integration tests to proof it works

Reply:
Gonna rephrase not to be rude sounding.
First thanks for the PR, and can you please explain to me why we need it here as it is already in the filename options. And could you please change the powertests.yaml file to be on point with the args, and could you also add either some tests cases in the generated tests dir using the shell script or directly write tests in the tests directory

Reply:
Sorry, I first implemented my change here but changed my mind. Missed this during commit review